packages/plans-grid-next/src/components/shared/plans-tooltip/index.tsx

```diff
@@ -61,25 +40,28 @@ export const Plans2023Tooltip = ( {
 
 	return (
 		<>
-			<HoverAreaContainer
-				className="plans-2023-tooltip__hover-area-container"
+			<span
+				className="plans-grid-next-tooltip__hover-area-container"
 				ref={ tooltipRef }
 				onMouseEnter={ () => ! isTouch && setActiveTooltipId( id ) }
 				onMouseLeave={ () => ! isTouch && setActiveTooltipId( '' ) }
 				onTouchStart={ () => isTouch && setActiveTooltipId( getMobileActiveTooltip() ) }
 				id={ id }
 			>
 				{ children }
-			</HoverAreaContainer>
-			<StyledTooltip
+			</span>
+			<Tooltip
+				className="plans-grid-next-tooltip"
 				isVisible={ isVisible }
 				position="top"
 				context={ tooltipRef.current }
 				hideArrow
 				showOnMobile={ showOnMobile }
 			>
 				{ text }
-			</StyledTooltip>
+			</Tooltip>
 		</>
 	);
 };
+
+export default PlansTooltip;
```

Comment:
I wonder if the component name could be plain `Tooltip` instead, or something that reflects what we do with the `Tooltip` import. WDYT? 

`PlansTooltip` certainly doesn't match the class added to it (`plans-grid-next-tooltip`). Should we create some convention around these?

Reply:
Yes I think the more generic `Tooltip` makes sense. [Updated](https://github.com/Automattic/wp-calypso/pull/93518/commits/415017c61086743631d0c220f7568789af6fc15c).

On the subject of naming, what was the idea behind `-next`? Is it a reference to this being the 'next' or upcoming version of the plans grid package?  Perhaps we can simplify and remove the `-next` everywhere?

Reply:
There's another `plans-grid` package doing similar work only a couple of years back in history.